som_crawlers/models/som_crawlers_task_step.py

```diff
@@ -260,9 +260,13 @@ def import_xml_files(self, cursor, uid, id, result_id, nivell=10, context=None):
         return output
 
     def recursive_extract_zip(self, zip_path, destination_path):
-        with zipfile.ZipFile(zip_path, "r") as zip_file:
-            zip_file.extractall(path=destination_path)
-        os.remove(zip_path)
+        try:
+            with zipfile.ZipFile(zip_path, "r") as zip_file:
+                zip_file.extractall(path=destination_path)
+        except:
```

Comment:
```suggestion
        except NOM_EXCEPCIÓ as e:
            import logging
            logger = logging.getLogger("openerp.som_crawlers")
            logger.info("Error al descomprimir zip: {}.format(e.msg()))
```

Jo faria algo així, posant els imports a dalt

Reply:
genial